app/components/govuk_component/summary_list_component.rb

```diff
@@ -1,20 +1,22 @@
 module GovukComponent
   class SummaryListComponent < GovukComponent::Base
-    attr_reader :borders, :actions, :card
+    attr_reader :borders, :actions, :card, :action_suffix
 
     renders_many :rows, ->(classes: [], html_attributes: {}, &block) do
       GovukComponent::SummaryListComponent::RowComponent.new(
         show_actions_column: @show_actions_column,
+        action_suffix: action_suffix || card&.title,
```

Comment:
I think I would be tempted to call the attribute `card_title` rather than `action_suffix`, just to be a bit clearer what you’d set it to? `action_suffix` might be a bit ambiguous, as it's not clear it's adding visually-hidden text, plus there’s already visually-hidden suffixes within the action links? 🤔 

```suggestion
    attr_reader :borders, :actions, :card, :card_title

    renders_many :rows, ->(classes: [], html_attributes: {}, &block) do
      GovukComponent::SummaryListComponent::RowComponent.new(
        show_actions_column: @show_actions_column,
        card_title: card_title || card&.title,
```

Reply:
I had it as `card_title` originally but it felt weird when rendering it standalone, so changed it. It feels weird to me that an argument called `card_title` doesn't set the title of a card.

Reply:
Hmm good point. I guess conceptually you're telling the summary list what the card_title _is_, so that it can be used as visually-hidden text. But maybe that's confusing...

Today in "naming things is hard"... 😂